core/src/navigation/ProductSwitcher.html

```diff
@@ -40,6 +60,36 @@
               on:click="{() => onActionClick(productSwitcherItem)}"
               data-testid="{getTestId(productSwitcherItem)}"
             >
+              {#if LuigiConfig.getConfigBooleanValue('navigation.addNavHrefs')}
+              <a
+                href="{addNavHrefForAnchor ? getRouteLink(productSwitcherItem) : undefined}"
```

Comment:
```suggestion
                href="{getRouteLink(productSwitcherItem)}"
```

**addNavHrefForAnchor** seems like a good idea, but might be redundant. 

I can see that **getRouteLink** also does the check too internally through **getNodeHref**. Seems like we added it long time ago:
getRouteLink -> RoutingHelpers.getNodeHref -> 

```
  getNodeHref(node, pathParams) {
    if (LuigiConfig.getConfigBooleanValue('navigation.addNavHrefs')) {
    ....
```

so maybe we could avoid this new variable here altogether.